packages/manager/modules/pci/src/projects/project/storages/object-storage/object-storage.routing.js

```diff
@@ -0,0 +1,66 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.storages.containers', {
+    url: '/containers',
+    component: 'pciProjectStorageObjectStorage',
```

Comment:
I don't think `/containers` should be under object storage section (also this means url change for customer right ?

Reply:
url doesnt change. Just the routes....may be i can add `storages.object-storage.objects.object` ?

Reply:
okay i can change it to `/object-storage` but url will change as we have a tab view now for object storage. Should url not change ?

Reply:
Has been refactored. Please have a look